addons/project/static/tests/tours/project_chatter_access_tour_tests.js

```diff
@@ -0,0 +1,64 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { stepUtils } from "@web_tour/tour_service/tour_utils";
+
+registry.category("web_tour.tours").add("project_chatter_log_disabled", {
+    test: true,
+    url: "/web",/*  */
+    steps: () => [
+        stepUtils.showAppsMenuItem(),
+        {
+            content: "Open Project app",
+            trigger: '.o_app[data-menu-xmlid="project.menu_main_pm"]',
+        },
+        {
+            content: "Toggle a Project's dropdown",
+            trigger: ".o_kanban_record .dropdown-toggle.o-no-caret",
+        },
+        {
+            content: "Wait for kanban actions to load",
+            trigger: ".dropdown-item.mx-0",
+        },
+        {
+            content: "Check that Log Note button is disabled",
+            trigger: ".o-mail-Chatter-logNote:disabled",
+            isCheck: true,
+        },
+        {
+            content: "Check that Log Note button is disabled",
+            trigger: ".o-mail-Chatter-sendMessage:disabled",
+            isCheck: true,
+        },
+    ],
+});
+
+registry.category("web_tour.tours").add("project_chatter_log_enabled", {
+    test: true,
+    url: "/web",/*  */
+    steps: () => [
+        stepUtils.showAppsMenuItem(),
+        {
+            content: "Open Project app",
+            trigger: '.o_app[data-menu-xmlid="project.menu_main_pm"]',
+        },
+        {
+            content: "Toggle a Project's dropdown",
+            trigger: ".o_kanban_record .dropdown-toggle.o-no-caret",
+        },
+        {
+            content: "Wait for kanban actions to load",
+            trigger: ".oe_kanban_action:contains('Settings')",
+        },    
```

Comment:
`content` string doesn't really match the selector intent. This is copy-paste from the `.dropdown-item` above.